src/libs/actions/PersonalDetails.js

```diff
@@ -100,6 +100,7 @@ function formatPersonalDetails(personalDetailsList) {
         const lastName = details.lastName || '';
         const payPalMeAddress = details.expensify_payPalMeAddress || '';
         const phoneNumber = details.phoneNumber || '';
+        const avatarHighResolution = details.avatar || OptionsListUtils.getDefaultAvatar(login);
```

Comment:
Why do we need this new property? And How come the default avatar is used here? we should depend on `details.avatar`. If a user has a default avatar, shouldn't it be available in `details.avatar`

Reply:
`const avatar = getAvatar(details, sanitizedLogin);`
```
function getAvatar(personalDetail, login) {
    if (personalDetail && personalDetail.avatarThumbnail) {
        return personalDetail.avatarThumbnail;
    }

    return OptionsListUtils.getDefaultAvatar(login);
}
```
avatar returned from `formatPersonalDetails` function uses avatarThumbnail and is of lower resolution for user uploaded images. So, new property is needed.

`const avatarHighResolution = details.avatar || details.avatarThumbnail;`

As `details.avatar` is undefined incase of default avatar. But, `details.avatarThumbnail` has the needed url. So, we can use `details.avatar || details.avatarThumbnail;`